l10n-pocket/en/pocket/tos.ftl

```diff
@@ -0,0 +1,128 @@
+
+pocket-tos-there-are-inherent = There are inherent risks in relying upon, using, or retrieving any information found on the internet, and we urge you to make sure you understand these risks before using the { -brand-name-pocket-technologies } or any website that provides you with access to the { -brand-name-pocket-technologies }. your use of the { -brand-name-pocket-technologies } is at your own risk. neither { -brand-name-mozilla } nor any of its parents, subsidiaries or affiliates, nor any of their employees, officers or directors, shall be liable for any direct, indirect, punitive, incidental, special, consequential, or other damages (including, without limitation, loss of data or information of any kind, loss of business, lost profits, interruption of business, cost of cover or any other damages) arising out of or in any way related to this agreement or the use or inability to use the { -brand-name-pocket-technologies }, whether based on contract, tort, strict liability or otherwise, even if we have been advised of the possibility of such damages. please note that some jurisdictions may not allow the exclusion of certain damages, so some of the above exclusions may not apply to you.
+
+pocket-tos-you-agree-at-your-own = You agree, at your own expense, to indemnify, defend and hold harmless { -brand-name-mozilla }, its parents, subsidiaries and affiliates, and their officers, directors, employees, agents, distributors and licensees, from and against any judgments, losses, deficiencies, damages, liabilities, costs, claims, demands, suits, and expenses (including, without limitation, reasonable attorneys’ fees, expert witness fees and expenses) incurred in, arising out of or in any way related to your breach of this Agreement, your use of the { -brand-name-pocket-technologies }, or any of your other acts or omissions.
+
+pocket-tos-any-dispute = Any dispute or claim arising out of or relating in any way to your use of any of the { -brand-name-pocket-technologies }, including without limitation any disputes related to the { -brand-name-pocket-premium } subscription service, will be resolved by binding arbitration, rather than in court, except that you may assert claims in small claims court if your claims qualify. the federal arbitration act and federal arbitration law apply to this agreement.
+pocket-tos-there-is-no-judge = There is no judge or jury in arbitration, and court review of an arbitration award is limited. an arbitrator, however, may award on an individual basis the same damages and relief as a court (including, without limitation, injunctive and declaratory relief or statutory damages), and must follow these terms of service as a court would. arbitration is more informal than a lawsuit in court, and allows for more limited discovery than in court. the arbitrator will decide all issues, including, without limitation, issues relating to the scope and enforceability of this arbitration provision.
```

Comment:
**Nit:** I know the CSS makes this sentence uppercase, but capitalize the first word after each period:
```suggestion
pocket-tos-there-is-no-judge = There is no judge or jury in arbitration, and court review of an arbitration award is limited. an arbitrator, however, may award on an individual basis the same damages and relief as a court (including, without limitation, injunctive and declaratory relief or statutory damages), and must follow these terms of service as a court would. Arbitration is more informal than a lawsuit in court, and allows for more limited discovery than in court. The arbitrator will decide all issues, including, without limitation, issues relating to the scope and enforceability of this arbitration provision.
```